incubator/elasticsearch/values.yaml

```diff
@@ -1,14 +1,10 @@
-## Elasticsearch cluster name
-## ref: https://github.com/bitnami/bitnami-docker-elasticsearch#environment-variables
-##
-name: elastic
-
-## Elasticsearch image
+## Bitnami Elasticsearch image version
 ## ref: https://hub.docker.com/r/bitnami/elasticsearch/tags/
+##
 image:
   registry: docker.io
   repository: bitnami/elasticsearch
-  tag: 5.5.2
+  tag: 6.2.3-r3
```

Comment:
rolling tag to`6.2.3` and pullPolicy to `Always`

Reply:
👍